### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nickheiner-usds**
Keeping things up to date and simplifying / clarifying where possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,23 +10,19 @@ Clerical errors have the potential to delay the resolution of a veteran's appeal
 [View application information on Caseflow Certification](https://github.com/department-of-veterans-affairs/caseflow/blob/master/docs/certification.md).
 
 ## Setup
+Install dependencies via Homebrew:
+
+    brew install postgresql redis chromedriver rbenv
```
Comment (shanear) [inline README.md]: love the one shot command haha

File: README.md [hunk 2]
```diff
@@ -25,23 +10,19 @@ Clerical errors have the potential to delay the resolution of a veteran's appeal
...
+    
 Make sure you have [rbenv](https://github.com/rbenv/rbenv) and [nvm](https://github.com/creationix/nvm) installed.
 
 Then run the following:
 
-    rbenv install 2.2.4
+    rbenv install <the version of Ruby defined in .travis.yml>
```
Comment (shanear) [inline README.md]: it'd be nice to still have an example here, just to show the formatting

> \# Replace 2.2.4 with the version of ruby defined in .travis.yml
> rbenv install 2.2.4

but either way is fine

File: README.md [hunk 3]
```diff
@@ -54,7 +35,7 @@ Install [pdftk](https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/pdftk_server
 Note this link was found on Stack Overflow and is not the same link that is on the pdftk website.
 The version on the website does not work on recent versions of OSX (Sierra and El Capitan).
 
-For the frontend, you'll need to install Node and the relevant npm modules
+For the frontend, you'll need to install Node and the relevant npm modules. [Install yarn](https://yarnpkg.com/en/docs/install). Use the version of Yarn defined in our `.travis.yml` file.
```
Comment (shanear) [inline README.md]: it'd be also nice to have the actual command to install yarn in here. I'm not too worried about it changing that much.

File: README.md (hunk 1 above)
Comment (nickheiner-usds) [reply README.md]: image

File: README.md (hunk 3 above)
Comment (nickheiner-usds) [reply README.md]: I think I'll keep it as-is. The install command does not change often, but people update docs even less frequently 😄.

**Pull request merged**